cosmos/constants.py

```diff
@@ -160,3 +160,7 @@ def _missing_value_(cls, value):  # type: ignore
 TESTABLE_DBT_RESOURCES = {DbtResourceType.MODEL, DbtResourceType.SOURCE, DbtResourceType.SNAPSHOT, DbtResourceType.SEED}
 
 DBT_COMPILE_TASK_ID = "dbt_compile"
+
+TELEMETRY_URL = "https://astronomer.gateway.scarf.sh/astronomer-cosmos/{telemetry_version}/{cosmos_version}/{airflow_version}/{python_version}/{platform_system}/{platform_machine}/{event_type}/{status}/{dag_hash}/{is_cosmos_dag}/{cosmos_task_groups_count}/{task_count}/{cosmos_task_count}"
+TELEMETRY_VERSION = "v1"
+TELEMETRY_TIMEOUT = 5.0
```

Comment:
For now, let's keep it to a second, and we can review / add a setting if needed:
```suggestion
TELEMETRY_TIMEOUT = 1.0
```